portal-backend/depmap/templates/tiles/sensitivity.html

```diff
@@ -1,11 +1,10 @@
-{% if dep_dists and dep_dists | length > 0 %}
+{% if dep_dist %}
```

Comment:
It seems like this tile, at one point in history, showed multiple distributions. However, that is no longer the case today (and hasn't been the case for a while), so I'm simplifying this implementation a bit.